data/repository/src/main/java/com/chocolate/repository/model/dto/direct_message/Chat.kt

```diff
@@ -0,0 +1,10 @@
+package com.chocolate.repository.model.dto.direct_message
+
+import java.util.Date
+
+data class Chat(
+    val id :String = "",
+    val secondMember:String = "",
```

Comment:
this is the second member's name or ID and what about renaming it to the receiver + name or ID